src/SmartFormat/Extensions/PluralLocalizationFormatter.cs

```diff
@@ -113,7 +113,7 @@ public bool TryEvaluateFormat(IFormattingInfo formattingInfo)
         // We can format numbers, and IEnumerables. For IEnumerables we look at the number of items
         // in the collection: this means the user can e.g. use the same parameter for both plural and list, for example
         // 'Smart.Format("The following {0:plural:person is|people are} impressed: {0:list:{}|, |, and}", new[] { "bob", "alice" });'
-        if (current is IConvertible convertible)
+        if (current is IConvertible convertible && current is not bool)
```

Comment:
We do still consider Enums which we did not previously although that seemed fine to me.

Reply:
Would be nice to have a unit test for that case 😉

Reply:
Guess, I oversaw an my mobile, that the test is already there.
Thanks a lot for submitting the fix.
(Merged current main, before the squash & merge.)